src/redux/actions/todo.actions.js

```diff
@@ -0,0 +1,13 @@
+import * as AT from '../action.types';
+
+export const changeDateRange = rangeDate => ({
+	type: AT.RANGE_DATE_REQUEST,
```

Comment:
keep the name of the action creator same as the action type
changeDateRange and CHANGE_DATE_RANGE